libs/ui-libraries/src/lib/ui-toolbar/ui-toolbar.component.scss

```diff
@@ -0,0 +1,42 @@
+@import '../../../../theme/theme.scss';
+
+mat-toolbar {
```

Comment:
Do we have the `bootstrap-utilities` and `bootstrap-4-grid` classes installed yet? Most of these styles can go away by leveraging those.